smurfs/src/reducers/index.js

```diff
@@ -0,0 +1,47 @@
+import { START_SMURF_FETCHING, FETCH_SMURF_SUCCESS, FETCH_SMURF_FAILURE, CREATE_SMURF_SUCCESS, CREATE_SMURF_FAILURE} from '../actions';
```

Comment:
ide group these in sections. a single line for all action creators of a grouping. just for readability